package.json

```diff
@@ -38,7 +39,7 @@
     "p-map": "^4.0.0",
     "tar": "^6.1.11",
     "tslib": "^2.0.0",
-    "uuid": "^8.3.2"
+    "yaml": "^1.10.2"
```

Comment:
oh it looks like we by accident removed `uuid`

Reply:
have added it back in, will commit next 👍